packages/cascara/src/modules/DataImage/DataImage.fixture.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import DataImage from './DataImage';
+import ModuleSandbox from '../ModuleSandbox';
+
+const DisplayDataImage = ({ isEditing, value, ...rest }) => (
```

Comment:
@rociobaena should we keep `value` in `...rest`?

Reply:
Definitely more simple to keep everything in rest except for the value that needs to come out to go to the provider